extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js

```diff
@@ -125,11 +125,10 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 			self._distributionDiv.html('');
 			self._pushtoResourceDiv.html('');
 			self.uri = self._baseUriSpan.text();
-			getFairCatalogs(self._baseUriSpan.text(), self);
-			return;
```

Comment:
Removing the `return` statement might have some side-effects?

Reply:
i do not foresee unexpected side effects which ones are you thinking off?